route-rs-runtime/src/composite/mod.rs

```diff
@@ -0,0 +1,4 @@
+/// Basic composite that take in M streams of a type, and outputs that same
+/// type to N packets.
+mod mton_composite;
```

Comment:
nit: Since the job of this link isn't to really transform anything, just take M streams of a type and output copies to N streams, can we call it `MultiTee` or something? `composite` doesn't really mean anything on its own, and in fact we might want to hide the fact that any given Link is a composite since that's an implementation detail.

Reply:
Hmm I like MultiClone, or some other spiffy name, I agree the composite it kind of an impl detail.